packages/shared/src/lib/featureManagement.ts

```diff
@@ -28,6 +32,7 @@ const feature = {
   showRoadmap: new Feature('show_roadmap', false),
   seoSidebar: new Feature('seo_sidebar', SeoSidebarExperiment.Control),
   onboardingChecklist: new Feature('onboarding_checklist', false),
+  upvote: new Feature('animated_upvote', UpvoteExperiment.Control),
```

Comment:
I would say let's make the first key also more explanatory.
eg: `animatedUpvote`

Reply:
```suggestion
  animatedUpvote: new Feature('animated_upvote', false),
```

@ilasw mind changing this to a boolean value 🙏 based on https://github.com/dailydotdev/apps/pull/3318#discussion_r1669892128 . I'll change the value in growthbook as well

Reply:
Done ✅

Reply:
Great I've changed the grwothbook value as well 🙌